### ISSUE TITLE
Temporary sandbox conversion on cluster vs. local

### ISSUE THREAD

**Issue by LukeLR** [apptainer/apptainer#737]
### Expected behavior

When running `apptainer shell apptainer.sif` on my local machine, the shell spawns instantly

### Actual behavior

When running the same command on our local HPC cluster, I get

```
INFO:    Converting SIF file to temporary sandbox...
```

causing a delay in spawning the shell

### Steps to reproduce this behavior

That's exactly my question: What causes this difference in behavior?

#### Cluster

```sh
$ cat /etc/os-release

NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

#### Local

```sh
$ cat /etc/os-release

NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
LOGO=archlinux-logo
```

### How did you install Apptainer

Installed from source in both cases, followed the instructions in [Install.md](https://github.com/apptainer/apptainer/blob/main/INSTALL.md)

**Comment by DrDaveD** [comment]
That sounds like pre-apptainer 1.1.0 behavior.  Can you run `apptainer --version` on your cluster to see if it is running the correct version?

**Comment by LukeLR** [comment]
Thanks for your reply! I usually had 1.0.3 installed and built 1.1.0 from source to see if this issue persists, both on the cluster and locally. `apptainer --version` shows indeed 1.1.0

**Comment by DrDaveD** [comment]
Do you have `squashfuse_ll` or `squashfuse` available on the cluster?  If you don't it should have shown with additional `INFO` messages.

**Comment by LukeLR** [comment]
No, squashfuse isn't present on the cluster, but it doesn't show additional `INFO` messages for it either.

**Comment by DrDaveD** [comment]
Hmm, on my system if `squashfuse` and `squashfuse_ll` are missing I see
```
INFO:    squashfuse not found, will not be able to mount SIF
FATAL:   container creation failed: mount /proc/self/fd/3->/var/apptainer/mnt/session/rootfs error: while mounting image /proc/self/fd/3: squashfuse not found
```

**Comment by DrDaveD** [comment]
Oh, and if `fuse2fs` is also missing I get
```
INFO:    squashfuse not found, will not be able to mount SIF
INFO:    fuse2fs not found, will not be able to mount EXT3 filesystems
INFO:    Converting SIF file to temporary sandbox...
```

**Comment by DrDaveD** [comment]
So I'm not sure what's going on in your case.  Clearly `squashfuse` or most preferably the multithreaded `squashfuse_ll` need to be available on the cluster, but I would still like to know how you could have the conversion without any `INFO` messages.  Please run it with `apptainer -vddd` and post the resultant debug messages.

**Comment by LukeLR** [comment]
Thanks a lot. So `squashfuse` is the only alternative to extracting the SIF? Or is there any other option?

**Comment by DrDaveD** [comment]
The only other option is compiling `--with-suid` (or installing the `apptainer-suid` package if you're using pre-built packages).

Could you please post that debugging output?

**Comment by AbdulAlkurdi** [comment]
I just installed Apptainer and faced the same sandbox issue. 

`apptainer exec lolcow_latest.sif cowsay moo
INFO:    squashfuse not found, will not be able to mount SIF
INFO:    fuse2fs not found, will not be able to mount EXT3 filesystems
INFO:    Converting SIF file to temporary sandbox...
INFO:    underlay of /etc/localtime required more than 50 (77) bind mounts
 _____
< moo >
 -----
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
INFO:    Cleaning up image...`
